src/telemetry/telemetry.go

```diff
@@ -22,12 +22,11 @@ var (
 	serverCert        = flag.String("server_crt", "", "TLS server certificate")
 	serverKey         = flag.String("server_key", "", "TLS server private key")
 	insecure          = flag.Bool("insecure", false, "Skip providing TLS cert and key, for testing only!")
-	jwtRefInt         = flag.Uint64("jwt_refresh_int", 30, "Seconds before JWT expiry the token can be refreshed.")
-	jwtValInt         = flag.Uint64("jwt_valid_int", 3600, "Seconds that JWT token is valid for.")
 )
 
 func main() {
 	flag.Var(userAuth, "client_auth", "Client auth mode(s) - cert,password,jwt")
+	flag.BoolVar(&gnmi.RbacDisable, "rbac-disable", false, "Enabled/Disable RBAC")
```

Comment:
Would be good to be consistent with `client_auth` and use underscore instead of dash

Reply:
changed